tests/test_writing_api.py

```diff
@@ -293,3 +293,24 @@ def test_validation_error_message_in_api_response(mocked_client):
 
     assert response.status_code == 422
     assert response.json()["detail"] == "a message"
+
+
+@patch("openapi_server.apis.writing_api.client_for_basic_auth")
+def test_unlock_on_fail(mocked_client):
+    # More generally testing errors are passed through from the client
+    mocked_client.return_value.save_locked_judgment_xml.side_effect = Mock(
+        side_effect=MarklogicResourceLockedError(),
+    )
```

Comment:
Might be worth adding a newline, to have the arrange - act - assert separation clearer.

Reply:
Done.